hudi-common/src/main/java/org/apache/hudi/common/table/view/AbstractTableFileSystemView.java

```diff
@@ -1030,7 +1030,7 @@ Stream<FileSlice> fetchLatestFileSlices(String partitionPath) {
    */
   Stream<FileSlice> fetchLatestFileSlicesBeforeOrOn(String partitionPath, String maxCommitTime) {
     return fetchAllStoredFileGroups(partitionPath)
-        .map(fileGroup -> fileGroup.getLatestFileSliceBeforeOrOn(maxCommitTime)).filter(Option::isPresent)
+        .map(fileGroup -> fileGroup.getLatestCompletedFileSliceBeforeOrOn(maxCommitTime)).filter(Option::isPresent)
```

Comment:
what is the impact of this change? what about other code paths that use this API?

Reply:
CI failed, seems have a side impact the other code paths, will fix it